src/plugins/charts/public/services/colors/colors.ts

```diff
@@ -44,7 +44,7 @@ import { seedColors } from './seed_colors';
 export class ColorsService {
   private _mappedColors?: MappedColors;
 
-  public readonly seedColors = seedColors;
+  public readonly seedColors = euiPaletteColorBlind();
```

Comment:
I wonder if we even need to provide this in the color service? Because consumers could also just get them directly from OUI. I suppose we can tackle that question once we do some of the other color refactoring and consolidation...

Reply:
@manasvinibs Can we make a follow-up issue to deprecate this public API and refactor the only known usage of it:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/fc4e696b0c32f71c886ea307038cbb6842389f7b/src/plugins/vis_type_tagcloud/public/plugin.ts#L57
and
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/fc4e696b0c32f71c886ea307038cbb6842389f7b/src/plugins/vis_type_tagcloud/public/components/tag_cloud_visualization.js#L61

as documented in:

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/fc4e696b0c32f71c886ea307038cbb6842389f7b/docs/charts/current_usage.md?plain=1#L14

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4406